orca_detector/orca_params.py

```diff
@@ -19,9 +19,13 @@ class DatasetType(IntEnum):
 OUTPUT_PATH = '/results/'
 WEIGHTS_PATH = '/vggish_weights/vggish_audioset_weights_without_fc2.h5'
 WEIGHTS_PATH_TOP = '/vggish_weights/vggish_audioset_weights.h5'
-OTHER_CLASS = 'Other'
+
 # Classification params
-REMOVE_CLASSES = []
+OTHER_CLASS = 'Other'
+REMOVE_CLASSES = ['BowheadWhale', 'Narwhal', 'SpermWhale',
+                  'TucuxiDolphin', 'White_sidedDolphin', 'DallsPorpoise',
+                 'HarpSeal', 'LeopardSeal', 'RibbonSeal',
```

Comment:
I checked, we still have a couple in

Reply:
Interesting, one of the BeardedSeal caused a lot of confusion to BowheadWhale. I wonder if we could exclude the BeardedSeal instead of the BowheadWhale

Reply:
Not all seals are excluded (I wondered that, too).  Check the [confusion matrix](https://docs.google.com/spreadsheets/d/1tSwioe0xXmBNbdb2fpGc8QDiSwEtnYPMwCvyiKBbesY/edit#gid=991221959) for examples.

I'm assuming we will iterate on this list, so I'd rather not block the PR on further EDA.